src/App.tsx

```diff
@@ -1,14 +1,62 @@
+import React from 'react';
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, View, Text } from 'react-native';
+import { registerRootComponent } from 'expo';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Navbar from './components/Navbar';
+import TrackerScreen from './screens/core/TrackerScreen';
+import SettingsScreen from './screens/core/SettingsScreen';
+import StatsScreen from './screens/core/StatsScreen';
+
+const Stack = createNativeStackNavigator();
+
+export default function App(): JSX.Element {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="Home" component={Navbar} options={{ headerShown: false }} />
+        <Stack.Screen name="Tracker" component={TrackerScreen} />
+        <Stack.Screen name="Settings" component={SettingsScreen} />
+        <Stack.Screen name="Stats" component={StatsScreen} />
+      </Stack.Navigator>
+      <StatusBar style="auto" />
```

Comment:
```suggestion
      <Navbar />
```

You actually don't need to do all of the Stack.Navigator stuff here. You already use Tab.Navigator in your Navbar.tsx file so you kind of wrote the same code twice if you know what I mean. So you can replace this block of code with just your component. It will clean up the code a lot.